eventmesh-common/src/main/java/org/apache/eventmesh/common/file/WatchFileTask.java

```diff
@@ -83,6 +78,11 @@ public void addFileChangeListener(FileChangeListener fileChangeListener) {
 
     public void shutdown() {
         watch = false;
+        try {
+            this.watchService.close();
+        } catch (IOException e) {
+            throw new RuntimeException("Unable to close WatchService", e);
+        }
```

Comment:
In your latest commit, you added the shutdown handling for the `watchService` in the `WatchFileTask#shutdown()` method. However, this doesn't actually change anything because the `shutdown()` method is only called before the application is closed. Nevertheless, I think the added code can still be kept.